text/chapter6.md

```diff
@@ -188,11 +188,11 @@ class Semigroup a where
   append :: a -> a -> a
 ```
 
-Strings form a semigroup under regular string concatenation, and so do arrays. Several other standard instances are provided by the `purescript-monoid` package.
+Strings form a semigroup under regular string concatenation, and so do arrays. Several other standard instances are provided by the `purescript-prelude` package.
```

Comment:
This should be `purescript-monoid`. `purescript-prelude` re-exports `Semigroup`, but it's not defined in there. There are those out there the prefer to define their own `Prelude`, so I think this should be changed back.

Reply:
Thanks for having a look. From what I have been able to gather is that `purescript-monoid` has been archived because the code has been merged into `purescript-prelude`. Maybe I'm missing something.

See: https://github.com/purescript-deprecated/purescript-monoid

```
$ spago install monoid
spago: The following packages do not exist in your package set:
  - monoid

$ spago install purescript-monoid
spago: The following packages do not exist in your package set:
  - purescript-monoid, and nor does `monoid`
```

Reply:
You're right! I didn't know that had happened.